test/integration/users_can_view_cars_by_location_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class UsersCanViewCarsByLocationTest < ActionDispatch::IntegrationTest
+  test 'user clicks location link and is routed to locations index' do
+    create_locations
+    visit "/"
+
+    within(".nav-wrapper") do
+      click_link "Locations"
+    end
+    assert_equal current_path, locations_path
+    assert page.has_content?("Locations")
+    assert page.has_content?("Capitol Hill")
+    assert page.has_content?("Highlands")
+    assert page.has_content?("Five Points")
+    assert page.has_content?("Lodo")
+  end
+
+  test 'user can view cars by location' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.